### DISCUSSION TITLE
怎样隐藏console窗口啊

### DISCUSSION THREAD

**Discussion by a563501734** [messense/aliyundrive-webdav#427] [Q&A] [answered]
我想配置个自动任务让系统在运行ddns任务时把它启动起来并映射网络驱动器，好让我从手机和电脑都能直接通过ipv6访问，但它老是打开一个黑框，真的有够碍眼的。。

**Comment by a563501734** [comment]
而且它弹出console窗口以后会阻塞自动任务的执行，先执行它就没法自动重新映射驱动器，先映射驱动器又映射不上

**Comment by messense** [comment, marked as answer]
参考 https://github.com/shadowsocks/shadowsocks-rust/issues/333

**Reply by a563501734** [reply]
谢谢了已经找到方法了：新建ps1脚本，写入以下代码
` Start-Process -FilePath ".\aliyundrive-webdav.exe" -ArgumentList "-rtoken -p8889 -Uuser -Wpwd --host ""::"" " -RedirectStandardError err.txt -RedirectStandardOutput log.txt -WindowStyle Hidden `
放在程序文件同目录中，点击执行即可
标准输出和错误输出分别重定向到log.txt和err.txt
如需停止
` Get-Process -name aliyundrive-webdav | kill `
